frame/collective/src/lib.rs

```diff
@@ -519,43 +565,67 @@ decl_module! {
 		///  - 3 mutations (`Voting`: codec `O(M)`, `ProposalOf`: codec `O(B)`, `Proposals`: codec `O(P2)`)
 		///  - any mutations done while executing `proposal` (`P1`)
 		/// - up to 3 events
-		/// - Benchmark: TODO
+		/// - Benchmarks:
+		///   - early disapproved: 37.21 + M * 0.239 + P2 * 0.466 + B * 0.002 µs (min squares analysis)
+		///   - early approved:    50.82 + M * 0.211 + P2 * 0.478 + B * 0.008 µs (min squares analysis)
+		///   - disapproved:       51.08 + M * 0.224 + P2 * 0.475 + B * 0.003 µs (min squares analysis)
+		///   - approved:          65.95 + M * 0.226 + P2 * 0.487 + B * 0.005 µs (min squares analysis)
 		/// # </weight>
-		#[weight = (200_000_000, DispatchClass::Operational)]
-		fn close(origin, proposal: T::Hash, #[compact] index: ProposalIndex) -> DispatchResultWithPostInfo {
+		#[weight = (
+			weight_for::close(
+				T::DbWeight::get(),
+				T::MaxMembers::get(), // `M`
+				*proposal_weight_bound, // `P1`
+				T::MaxProposals::get() // `P2`
+			),
+			DispatchClass::Operational
+		)]
+		fn close(
+			origin, proposal: T::Hash, #[compact] index: ProposalIndex, proposal_weight_bound: Weight
+		) -> DispatchResultWithPostInfo {
 			let _ = ensure_signed(origin)?;
-
+		
```

Comment:
Not sure the suggestion works correctly but my point is to remove whitespace not the line
```suggestion

```